core/src/main/scala/spire/math/Number.scala

```diff
@@ -609,7 +609,14 @@ private[math] case class RationalNumber(n: Rational) extends Number { lhs =>
 }
 
 trait NumberInstances {
-  implicit final val NumberAlgebra = new NumberAlgebra
+  implicit final val NumberAlgebra: Field.WithDefaultGCD[Number]
```

Comment:
Why isn't this `implicit final val NumberAlgebra: NumberAlgebra = new NumberAlgebra`?

Reply:
I'm following the style of cats like:
https://github.com/typelevel/cats/blob/main/kernel/src/main/scala/cats/kernel/instances/BigIntInstances.scala#L5

Reply:
Fair enough ... I don't know why it's done that way there.